spring-cloud-netflix-core/pom.xml

```diff
@@ -199,4 +199,25 @@
 			<optional>true</optional>
 		</dependency>
 	</dependencies>
+	<profiles>
+		<profile>
+			<id>java8plus</id>
+				<activation>
+					<jdk>[1.8,2.0)</jdk>
```

Comment:
I went to some lengths not to break Java 7 compatibility, hence the reflection-based strategy for enabling the related unit test. That Maven profile is enabled when running on JDK >= 1.8 and in that case compiles with the -parameters argument, but this doesn't seem to create problematic bytecode. My assumption is that the Java 7 bytecode structure already included whatever mechanism is used in Java 8 to hold the parameter names.